doc/generated_testcases_v2.yaml

```diff
@@ -0,0 +1,221 @@
+# Example generators.yaml
+
+# The solution is used to generate a .ans for each generated .in which doesn't
+# yet have a corresponding .ans. If there are generators that don't write a .ans
+# file themselves, this must be specified.
+# This should read the input from stdin and write to stdout.
+#
+# Absolute paths are taken relative to the problem root, relative paths are
+# taken relative to the generators directory.
+#
+# TOOLING: may pick a default if not specified, but should raise a warning.
+solution: /submissions/accepted/sol.py
+
+# The visualizer is used when no suitable image was generated already.
+# This should read {name}.in and/or {name}.ans from the current working
+# directory, and write {name}.ext for an extension in:
+# .png, .jpg, jpeg, .svg
```

Comment:
Nitpick: why keep both `.jpg` and `.jpeg`? I would stick to `.jpg` unless there is a clear reason to also support `.jpeg`.

Reply:
Ok, good to keep it the same as there, although I'd say that maybe there also `.jpeg` should be dropped.